shims/spark312/src/main/scala/com/nvidia/spark/rapids/shims/spark312/Spark312Shims.scala

```diff
@@ -34,22 +34,6 @@ class Spark312Shims extends SparkBaseShims {
 
   override def hasCastFloatTimestampUpcast: Boolean = true
 
-  override def int96ParquetRebaseRead(conf: SQLConf): String = {
```

Comment:
these are still defined in 311 and 313 Shims.scala, I assume they can be removed?